services/vrf/vrf_test.go

```diff
@@ -0,0 +1,130 @@
+package vrf
```

Comment:
🙌 Love that these tests are all in the same package. Would be great to move towards this more often.